projects/packages/my-jetpack/_inc/components/product-interstitial/index.jsx

```diff
@@ -71,6 +72,7 @@ export default function ProductInterstitial( {
 	const { recordEvent } = useAnalytics();
 	const { onClickGoBack } = useGoBack( { slug } );
 	const { myJetpackCheckoutUri = '' } = getMyJetpackWindowInitialState();
+	const { siteIsRegistering, handleRegisterSiteOnly } = useConnection();
```

Comment:
If you pass `skipUserConnection: true` to `useConnection` you'll be able to achieve a similar result to `handleRegisterSiteOnly`, since that property skips the connection in the `handleConnectUser` method.

Reply:
Yes, good call. That would simplify this PR a ton :)

I'll make updates with this approach 👍 

Thanks!